magic/magic.go

```diff
@@ -30,8 +30,6 @@ func LoadMagic() MagicMap {
 	magicMap.ByValue[1631600659] = "CPUINFO"
 	magicMap.ByName["CREDS"] = 1409430855
 	magicMap.ByValue[1409430855] = "CREDS"
-	magicMap.ByName["CRTOOLS_IMAGES_V1_1"] = 2
-	magicMap.ByValue[2] = "CRTOOLS_IMAGES_V1_1"
```

Comment:
Why are these two lines being removed?

Reply:
Ah, the regex parsing does not match non-hex values, so these lines are left out from the new run. Would we need these though? They are not image types themselves, so I'd assume they're not directly used by go-criu?

Reply:
[`CRTOOLS_IMAGES_V1_1`](https://github.com/checkpoint-restore/criu/blob/c2f9f900a94cbfefda9385f7dd1b757b226bbaa2/criu/include/magic.h#L13) is used as a common magic for protobuf all [images](https://criu.org/Images) except inventory and service files:
```
IMAGE_FILE ::= MAGIC [MAGIC_2] { ENTRY }
```

See https://github.com/checkpoint-restore/criu/commit/f5ea330ce1f38486a9f80d68504899d7dad48cb3